stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt

```diff
@@ -43,6 +54,27 @@ public class MessageComposerController(
      */
     public val selectedAttachments: MutableStateFlow<List<Attachment>> = MutableStateFlow(emptyList())
 
+    /**
+     * Represents the maximum allowed message length.
+     */
+    public val maxMessageLength: MutableStateFlow<Int> = MutableStateFlow(Integer.MAX_VALUE)
+
+    /**
+     * Sets up the data loading operations such as observing the maximum allowed message length.
+     */
+    init {
+        scope.launch {
+            val result = chatDomain.watchChannel(channelId, 0).await()
+
+            if (result.isSuccess) {
+                val channelController = result.data()
+                channelController.channelConfig.collect {
+                    maxMessageLength.value = it.maxMessageLength
```

Comment:
Will this be updated in the runtime, based on the dashboard config? If not, we can just collect it once, update the value and then stop collecting.

That might help us as we won't have to collect the values and cancel/clean up the scoep.

Reply:
Unfortunately, I anticipate some other use cases like observing channel members for suggestions, where we will need to observe the changes. However, you are right here. Support for updating this value in real time not needed.

Reply:
Also, the state in the controller is not updated instantly:
`internal val channelConfig: MutableStateFlow<Config> = MutableStateFlow(Config())`
Thinking about a hack which implies subscribing to something else that 100% happens after the network request and then collecting the channel config value.